### PR TITLE
Workday long running - Testing

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30112

### CODE REVIEW

**Comment by Shellyber** [comment]
@dantavori Updated according to our conversion.

**Review by dantavori — APPROVED** [review]
please replace the reports to the updated ones (you can get them from the dev instance)

**Comment by Shellyber** [comment]
@dantavori Please have a look also on the README file.
If you have any more inputs/use-cases to add.

**Comment by Shellyber** [comment]
@kirbles19 Please review the README file. Especially the first part where the explanation is happening.

File: Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+IAM mock Workday full reports and events.
+
+This integration is for testing propose only.
+To use this integration for testing, take the server with the listening port that has been configured in the integration settings,
+and configure the **Workday_IAM** instance with this url and invoke the *fetch_incidents*.
```
Comment (dantavori) [inline Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md:5]: looks like you covered almost everything, please just add a note to mark the `longRunning` checkbox as well

**Comment by Shellyber** [comment]
related to https://github.com/demisto/etc/issues/31369

File: Packs/Workday/Integrations/WorkdayIAMEventsGenerator/WorkdayIAMEventsGenerator.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  type: 8
+- display: Port mapping (<port> or <host port>:<docker port>)
+  name: longRunningPort
+  required: false
+  type: 0
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+description: IAM mock Workday full reports and events.
```
Comment (anara123) [inline Packs/Workday/Integrations/WorkdayIAMEventsGenerator/WorkdayIAMEventsGenerator.yml:18]:
```suggestion
description: Generates Workday dummy test reports and events. Used for testing/dev purposes.
```

File: Packs/Workday/Integrations/WorkdayIAMEventsGenerator/WorkdayIAMEventsGenerator.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  type: 8
+- display: Port mapping (<port> or <host port>:<docker port>)
+  name: longRunningPort
+  required: false
+  type: 0
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+description: Generates Workday dummy test reports and events. Used for testing/dev
```
Comment (kirbles19) [inline Packs/Workday/Integrations/WorkdayIAMEventsGenerator/WorkdayIAMEventsGenerator.yml:18]: I think we should stick with "mock".

File: Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+Generates Workday dummy test reports and events. Used for testing/dev purposes.
+
+To use this integration for testing:
+1. Create an instance - Add a listening port and mark the *Long running instance* checkbox.
```
Comment (kirbles19) [inline Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md:4]: 1. Create an instance.
   a. Add a listening port
   b. Select the **Long running instance** checkbox.

File: Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+2. Take the server with the listening port that has been configured in the integration settings,
+and configure the **Workday_IAM** instance with this url and invoke the *fetch_incidents*.
```
Comment (kirbles19) [inline Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md:8]: 2. Take the server URL and listening port that you entered in the integration settings and configure the **Workday_IAM** instance and invoke *fetch_incidents*. 

@Shellyber maybe we should put an example of the URL and port? I know it's basic, but couldn't hurt.

File: Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+For more information about how to configure the long running integration go to [Long Running HTTP Integrations](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke).
+
+
+This integration uses mock reports (simulates the real reports that are coming from Workday).
```
Comment (kirbles19) [inline Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md:13]: This integration uses mock reports (simulates the real reports from Workday).

File: Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+You can use the **generate-event** command to get all kinds of reports (hire, update, terminate and rehire)
```
Comment (kirbles19) [inline Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md:14]: You can use the **generate-event** command to get all kinds of reports (hire, update, terminate, and rehire).

File: Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
+Generates Workday dummy test reports and events. Used for testing/dev purposes.
```
Comment (kirbles19) [inline Packs/Workday/Integrations/WorkdayIAMEventsGenerator/README.md]: Generates mock Workday reports and events, which you should use for testing and development purposes.

File: Packs/Workday/ReleaseNotes/1_0_6.md [hunk 9]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### New: Workday IAM Event Generator (Beta)
+- IAM mock Workday full reports and events. (Available from Cortex XSOAR 6.0.0).
```
Comment (kirbles19) [inline Packs/Workday/ReleaseNotes/1_0_6.md:4]: This integration generates mock Workday reports and events, which you should use for testing and development purposes. (Available from Cortex XSOAR 6.0.0).

**Review by kirbles19 — APPROVED** [review]

**Pull request merged**